recipes-core/initrdscripts/initramfs-framework_%.bbappend

```diff
@@ -0,0 +1,56 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI += " \
+    file://lvm \
+    file://bootfs \
+    file://tpm \
+    file://tpm2 \
+    file://selinux \
+    "
+
+do_install_append() {
+    install -d ${D}/init.d
+
+    # lvm
```

Comment:
The comments here aren't particularly informative.

Reply:
I was just following what was done in the upstream recipe.